admin/python/lsst/qserv/admin/configure.py

```diff
@@ -369,8 +227,149 @@ def has_configuration_step(steps):
     """
     return bool(intersect(steps, CONFIGURATION_STEPS))
 
+class Templater(object):
+
+    def __init__(self, qserv_version):
+
+        self.qserv_version = qserv_version
+        self._templateParams = None
+
+    def _setPerms(self, file):
+        (path, basename) = os.path.split(file)
+        script_list = [c + ".sh" for c in COMPONENTS]
+        if (os.path.basename(path) == "bin" or
+                    os.path.basename(path) == "init.d" or
+                    basename in script_list):
+            os.chmod(file, 0o760)
+        elif basename in SECRET_FILES:
+            os.chmod(file, 0o600)
+        else:
+            # all other files are configuration files
+            os.chmod(file, 0o660)
+
+    def _initTemplateParams(self):
+        """ Compute templates parameters from Qserv meta-configuration file
+            from PATH or from environment variables for products not needed during build
+        """
+
+        if self._templateParams is None:
+
+            config = commons.getConfig()
+
+            if config['qserv']['node_type'] == 'mono':
+                comment_mono_node = '#MONO-NODE# '
+            else:
+                comment_mono_node = ''
+
+            scisql_dir = os.environ.get('SCISQL_DIR')
+            if scisql_dir is None:
+                _LOG.fatal(
+                    "sciSQL install : sciSQL is missing, please install it and set SCISQL_DIR environment variable.")
+                sys.exit(1)
+
+            # find python executable in $PATH
```

Comment:
Does the python have to be in the path or can it be the python that is running this code?
```python
python_bin = sys.executable
```
would be simpler.